tools/FindUnity/Program.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+
+namespace FindUnity
+{
+    internal static class Program
+    {
+        // If SpecialFolder.ProgramFiles returns the (x86) variant instead of "Program Files",
+        // check that "Prefer 32 bit" is unticked in the project's Build settings.
+        private static readonly string ProgramFiles = Environment.GetFolderPath(Environment.SpecialFolder.ProgramFiles);
+        private static readonly string UnityHubPath = Path.Combine(ProgramFiles, "Unity", "Hub");
+        private static readonly string UnityPath = Path.Combine(ProgramFiles, "Unity");
+        private static readonly string ImprobableUnityPath = Path.Combine(@"c:\", "Unity");
```

Comment:
this will only work for windows. Could we already add the support for mac / paths for it as I'm currently checking everything around it

Reply:
Yep, I'll do that tomorrow when I have my Mac again :)